apps/doc-gen/src/app/App.tsx

```diff
@@ -29,7 +29,7 @@ const App = () => {
 
   return (
     <div className="p-3">
-      <h5 className="h-5 mb-3">Compile a solidity contract in order to build documentation as markdown or right click on a contract and click on "Generate Documentation".</h5>
+      <h5 className="h-5 mb-3">Compile a Solidity contract and generate its documentation as Markdown. (Shortcut: right-click on a contract in the File Explorer and select "Generate Docs" from the context menu.).</h5>
```

Comment:
details you have added in not a `shortcut`

Reply:
If you check the dev-ux channel in discord, you will understand why the wording has changed.